cernopendata/modules/fixtures/data/articles/cms-guide-for-research/research.md

```diff
@@ -24,65 +24,65 @@ If you are interested in conducting a more sophisticated analysis using CMS Open
-- If you want to try both datasets, you will need to use [the appropriate VMs](http://opendata-dev.web.cern.ch/search?page=1&size=20&tags=VM&experiment=CMS).
+- If you want to try both datasets, you will need to use [the appropriate VMs](/search?page=1&size=20&tags=VM&experiment=CMS).
 
 <center>–––</center>
 
 > "I want to install the CMS software environment on the virtual machine (separately for 2010 and 2011/2012 data) which is needed for access to and analysis of CMS Research level data."
 
-- Install [the approriate VM](http://opendata-dev.web.cern.ch/search?page=1&size=20&tags=VM&experiment=CMS), 2010 VM for 2010 data, and 2011 VM for 2011/2012 data.
-- Go to "Getting started with CMS data:   ["Getting started with CMS 2010 data"](http://opendata-dev.web.cern.ch/articles/getting-started-with-cms-open-data) for 2010 or ["Getting started with CMS 2011 data"](http://opendata-dev.web.cern.ch/articles/getting-started-with-cms-2011-data) for 2011/2012 data FIXME check links
+- Install [the approriate VM](/search?page=1&size=20&tags=VM&experiment=CMS), 2010 VM for 2010 data, and 2011 VM for 2011/2012 data.
+- Go to: ["Getting started with CMS 2010 data"](/articles/getting-started-with-cms-2010-data) for 2010 or ["Getting started with CMS 2011 data"](/articles/getting-started-with-cms-2011-data) for 2011/2012 data FIXME check links
 
-**Note**: The 2010 (SL5) virtual machine will only work on 2010 data with CMSSW 4-2-8 (and other SLC5 based CMSSW releases). The 2011 (SL6) virtual machine will only work on 2011/2012 data and MC with CMSSW 5-3-32 (and other SLC6 based CMSSW releases).
+**Note**: The 2010 (SL5) virtual machine will only work on 2010 data with CMSSW 4-2-8 (and other SLC5-based CMSSW releases). The 2011 (SL6) virtual machine will only work on 2011/2012 data and MC with CMSSW 5-3-32 (and other SLC6-based CMSSW releases).
 
 <center>–––</center>
 
 > "I want to produce some example physics distributions (e.g. inclusive di-muon spectrum analysis example directly from AOD or two-/four-lepton analysis example with intermediate ntuples)."
 
 - Install the CMS software as in the previous item (for 2010 or 2011).
-- Follow options A (di-muon spectrum, simpler) or B (two-/four-lepton example, more sophisticated). For 2011 option A, take the SingleMu or DoubleMu dataset index files from [http://opendata.cern.ch/record/32](http://opendata.cern.ch/record/32) or [http://opendata.cern.ch/record/17](http://opendata.cern.ch/record/17).
+- Follow options A (di-muon spectrum, simpler) or B (two-/four-lepton example, more sophisticated)in [Getting started with CMS 2010 data](/articles/getting-started-with-cms-2010-data). For 2011 option A, take the SingleMu or DoubleMu dataset index files from [http://opendata.cern.ch/record/32](/record/32) or [http://opendata.cern.ch/record/17](/record/17).
```

Comment:
- There's a space missing after the second set of brackets.
- `[Getting started with CMS 2010 data]` should probably be `["Getting started with CMS 2010 data"]`.